federatedscope/core/workers/server.py

```diff
@@ -901,6 +926,27 @@ def eval(self):
             self.broadcast_model_para(msg_type='evaluate',
                                       filter_unseen_clients=False)
 
+    def _calculate_model_delta(self, init_model, updated_model):
+        model_delta = copy.deepcopy(self.model.state_dict())
+        for key in self.model.state_dict():
+            model_delta[key] = updated_model[key] - init_model[key]
+        return model_delta
+
+    def _global_trainer(self):
```

Comment:
If it is only used for `Fltrust aggregator`, I think it should not be placed in `server.py`